Resources/ConfigPresets/WizardsDen/salamander.toml

```diff
@@ -11,6 +11,7 @@ rules_header = "ui-rules-header-rp"
 [whitelist]
 enabled = true
 reason = "whitelist-not-whitelisted-rp"
+min_players = 5
```

Comment:
5 is hilariously low, make it at least 20.

Reply:
i'm happy to make it 10 but considering sala highpop is 20-25 i wouldn't prefer to do so

Reply:
20 is still considered lowpop in ss13.